# Generated by Django 2.1.8 on 2019-09-09 23:46

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('balance_scorecard', '0026_auto_20190909_1940'),
    ]

    operations = [
        migrations.CreateModel(
            name='Compromisos',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('correcion_incidente', models.CharField(max_length=100)),
                ('fecha_plazo', models.DateField()),
                ('estado_compromiso', models.SmallIntegerField(choices=[(1, 'Completada'), (2, 'Pendiente'), (3, 'En Proceso')])),
                ('responsable', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
            ],
        ),
        migrations.CreateModel(
            name='Incidentes',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('titulo_incidente', models.CharField(max_length=40)),
                ('fecha_incidente', models.DateTimeField(auto_now_add=True)),
                ('causa_incidente', models.CharField(blank=True, max_length=100, null=True)),
                ('indicadores_afectados', models.ManyToManyField(to='balance_scorecard.Indicador')),
                ('registrador', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
            ],
        ),
        migrations.CreateModel(
            name='Reunion',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('fecha_reunion', models.DateField()),
                ('participantes', models.ManyToManyField(to=settings.AUTH_USER_MODEL)),
            ],
        ),
    ]
